specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2021-10-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2275 @@
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "Resource system metadata."
+        }
+      }
+    },
+    "ConfigurationStoreProperties": {
+      "description": "The properties of a configuration store.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state of the configuration store.",
+          "enum": [
+            "Creating",
+            "Updating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "creationDate": {
+          "format": "date-time",
+          "description": "The creation date of configuration store.",
+          "type": "string",
+          "readOnly": true
+        },
+        "endpoint": {
+          "description": "The DNS endpoint where the configuration store API will be available.",
+          "type": "string",
+          "readOnly": true
+        },
+        "encryption": {
+          "$ref": "#/definitions/EncryptionProperties",
+          "description": "The encryption settings of the configuration store."
+        },
+        "privateEndpointConnections": {
+          "description": "The list of private endpoint connections that are set up for this resource.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionReference"
+          }
+        },
+        "publicNetworkAccess": {
+          "description": "Control permission for data plane traffic coming from public networks while private endpoint is enabled.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PublicNetworkAccess",
+            "modelAsString": true
+          }
+        },
+        "disableLocalAuth": {
+          "type": "boolean",
+          "default": false,
+          "description": "Disables all authentication methods other than AAD authentication."
+        },
+        "softDeleteRetentionInDays": {
+          "type": "integer",
+          "format": "int32",
+          "default": 7,
+          "description": "The amount of time in days that the configuration store will be retained when it is soft deleted."
+        },
+        "enablePurgeProtection": {
+          "type": "boolean",
+          "default": false,
+          "description": "Property specifying whether protection against purge is enabled for this configuration store."
+        },
+        "createMode": {
+          "type": "string",
+          "description": "Indicates whether the configuration store need to be recovered.",
+          "enum": [
```

Comment:
@x9xu  Can you capitalize the first letter of each? All of our enums use capital on first letter.

Reply:
```
          "enum": [
            "Recover",
            "Default"
          ],
```